<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>788</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ATLAS</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="2" column="0">
     <widget class="QFrame" name="NavFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="2" column="1">
        <widget class="QFrame" name="HomeFrame">
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QToolButton" name="homeButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>19</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:none;
background-color: transparent;
margin-left: 2px;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/images/assets/Atlas_logo_v2.svg</normaloff>:/images/assets/Atlas_logo_v2.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="SearchIcon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0,40);
border:none;
margin-left:4px;
border-top-left-radius:5px;
border-bottom-left-radius:5px;</string>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resources.qrc">:/images/assets/search.svg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>6</number>
            </property>
            <property name="indent">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="SearchBox">
            <property name="minimumSize">
             <size>
              <width>132</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0,40);
border:none;
border-top-right-radius:0px;
border-bottom-right-radius:0px;
border-bottom-left-radius:0px;
border-top-left-radius:0px;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0,40);
border:none;
border-top-right-radius:5px;
border-bottom-right-radius:5px;
border-bottom-left-radius:0px;
border-top-left-radius:0px;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="sortSelection">
            <item>
             <property name="text">
              <string>Name</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Creator</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Engine</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Import Time</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="sortOrderButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>42</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>ASC</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QTreeView" name="gamesTree"/>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>4</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QFrame" name="StatusFrame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>36</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="AddGame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Add Game</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/assets/addgame.svg</normaloff>:/images/assets/addgame.svg</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="GamesInstalled">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0 Games Installed</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QWidget" name="NavTop" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(50,50,50,100);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QToolButton" name="homeButton_top">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>19</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:none;
background-color: transparent;
margin-left: 2px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/assets/Atlas_logo_v2.svg</normaloff>:/images/assets/Atlas_logo_v2.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0,40);
border:none;
margin-left:4px;
border-top-left-radius:5px;
border-bottom-left-radius:5px;</string>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources.qrc">:/images/assets/search.svg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>171</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0,40);
border:none;
border-top-right-radius:5px;
border-bottom-right-radius:5px;
border-bottom-left-radius:0px;
border-top-left-radius:0px;
margin-right:5px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="bv">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="RecordListView" name="recordView">
          <property name="movement">
           <enum>QListView::Free</enum>
          </property>
          <property name="flow">
           <enum>QListView::LeftToRight</enum>
          </property>
          <property name="isWrapping" stdset="0">
           <bool>true</bool>
          </property>
          <property name="resizeMode">
           <enum>QListView::Adjust</enum>
          </property>
          <property name="layoutMode">
           <enum>QListView::SinglePass</enum>
          </property>
          <property name="uniformItemSizes">
           <bool>true</bool>
          </property>
          <property name="batchSize">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="cv">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>28</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="horizontalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="GameWidget" name="detailedRecordView" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1043</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionListView"/>
    <addaction name="actionBannerView"/>
    <addaction name="actionCoverView"/>
    <addaction name="separator"/>
    <addaction name="actionArrangeBy"/>
   </widget>
   <widget class="QMenu" name="menuGames">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionSimpleImporter"/>
     <addaction name="actionBulkImporter"/>
     <addaction name="actionGameListImporter"/>
     <addaction name="actionSingleImporter"/>
    </widget>
    <addaction name="menuImport"/>
    <addaction name="actionMange"/>
    <addaction name="actionDownload"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionUpdates"/>
    <addaction name="separator"/>
    <addaction name="actionAboutAtlas"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionViewFileHistory"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuGames"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionMange">
   <property name="text">
    <string>Mange</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="text">
    <string>Download</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionListView">
   <property name="text">
    <string>List View</string>
   </property>
  </action>
  <action name="actionBannerView">
   <property name="text">
    <string>Banner View</string>
   </property>
  </action>
  <action name="actionCoverView">
   <property name="text">
    <string>Cover View</string>
   </property>
  </action>
  <action name="actionArrangeBy">
   <property name="text">
    <string>Arrange By</string>
   </property>
  </action>
  <action name="actionUpdates">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionAboutAtlas">
   <property name="text">
    <string>About Atlas</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionViewFileHistory">
   <property name="text">
    <string>View file history</string>
   </property>
  </action>
  <action name="actionSimpleImporter">
   <property name="text">
    <string>Simple</string>
   </property>
  </action>
  <action name="actionBulkImporter">
   <property name="text">
    <string>Bulk</string>
   </property>
  </action>
  <action name="actionGameListImporter">
   <property name="text">
    <string>GameList</string>
   </property>
  </action>
  <action name="actionSingleImporter">
   <property name="text">
    <string>Single</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/games/GameWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RecordListView</class>
   <extends>QListView</extends>
   <header>ui/views/RecordListView.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
